literature/literature.bib

```diff
@@ -27541,6 +27566,14 @@ @book{BS03
 	isbn      = {3540007024},
 }
 
+@book{DF03,
+	author    = {David S. Dummit and Richard M. Foote},
+	title     = {{Abstract Algebra}},
+	publisher = wiley,
+	year      = 2003,
+	ektags    = {compositionality},
```

Comment:
address missing?

Reply:
there is no address.

Reply:
This knowledge should be documented via `missing = noaddress` then.

Reply:
I am not sure I agree, none of our validations require an address field, and I have never seen any standard requiring this or `.bst` file using it. We also barely use it (21 times).

Nevertheless, i'll add these two in my next pull request.

Reply:
Oh. You defeated me. We were trusting a random website and @tthuem . Thank you for adding the two `missing` fields for pseudo-consistency anyway. :sweat_smile: There seems to be a misunderstanding in our team and the BibTags documentation then.